user-contracts-mon/main.go

```diff
@@ -0,0 +1,39 @@
+package main
+
+import (
+	"flag"
+	"log"
+	"os"
+
+	"github.com/NicoNex/echotron/v3"
+	"github.com/threefoldtech/tfgrid-sdk-go/grid-client/deployer"
+	monitor "github.com/threefoldtech/tfgrid-sdk-go/user-contracts-mon/internal"
+)
+
+func main() {
```

Comment:
create StartApp function that does creation of the monitor, the tfplugin, and the pulling (and returns the error) and the only code we should have in main is 

if err := StartApp(); err != nil {

    log.Fatal....
}